player/webpack.config.js

```diff
@@ -16,12 +16,28 @@ const CopyWebpackPlugin = require('copy-webpack-plugin');
 const ImageminPlugin = require('imagemin-webpack-plugin').default;
 const CleanWebpackPlugin = require('clean-webpack-plugin');
 const ngAnnotatePlugin = require('ng-annotate-webpack-plugin');
+const fs = require('fs');
+const replace = require('replace-in-file');
+const file_extra = require('fs-extra')
+var WebpackOnBuildPlugin = require('on-build-webpack');
+
 
 const APP_CONFIG = {
-    ekstep: './public/js/appConfig.js',
-    sunbird: './public/js/appConfig-Sunbird.js'
+    ekstep: {
```

Comment:
Create new build.config.json for webpack build configuration purpose. So that any other adopters can change only build.config.json for build time changes.